docs/installing-solidity.rst

```diff
@@ -6,6 +6,28 @@
 Installing Solidity
 ###################
 
+Versioning
+==========
+
+Solidity versions follow `semantic versioning <https://semver.org>` and in addition to
+releases, **nightly development builds** are also made available.  The nightly builds
+are not guaranteed to be working and despite best efforts they might contain undocumented
+and/or broken changes.
+
+After a release is made, the patch version level is bumped, because we assume that only
+patch level (non-breaking and non-feature) changes follow. When changes are merged, the
```

Comment:
Right, maybe we need to make it much clearer that bumping happens according to semver.